frontend/src/metabase/common/components/DatePicker/DatePicker.tsx

```diff
@@ -87,3 +90,35 @@ export function DatePicker({
       );
   }
 }
+
+interface SimpleDatePickerProps {
+  value?: DatePickerValue;
+  availableOperators?: ReadonlyArray<DatePickerOperator>;
+  onChange: (value: DatePickerValue | undefined) => void;
+}
+
+export function SimpleDatePicker({
+  value: initialValue,
+  availableOperators = DATE_PICKER_OPERATORS,
+  onChange,
+}: SimpleDatePickerProps) {
```

Comment:
nit: I think it'd be more intuitive to have a `SimpleDatePicker` file

Reply:
Agree, fixed